indra/db/util.py

```diff
@@ -541,10 +546,9 @@ def _get_filtered_rdg_statements(db, get_full_stmts, clauses=None,
                                           clauses)
 
     # Specify sources of fulltext content, and order priorities.
-    full_text_content = ['manuscripts', 'pmc_oa', 'elsevier']
+    full_text_content = ['manuscripts', 'pmc_oa', 'elsevier', 'pubmed']
```

Comment:
What kind of content does `pubmed` refer to in this context?

Reply:
This would be pubmed abstracts.